lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -0,0 +1,35 @@
+const Node = require('../node');
+
+// Returns true if by adding a term you can simplify part of the function into an integer
```

Comment:
I know it's almost 80 characters here so it feels annoying to put it in two lines, but... 80 character max please ^_^